src/senaite/patient/content/patient.py

```diff
@@ -329,6 +329,15 @@ class IPatientSchema(model.Schema):
         required=False,
     )
 
+    age = schema.TextLine(
+        title=_(u"label_patient_age", default=u"Age"),
+        description=_(
+            u"description_patient_age",
+            default=u"Age in YMD format or a single number representing years"
```

Comment:
Although the input field accepts a value in `ymd` format, it does not accept a year